packages/js/product-editor/src/blocks/product-fields/summary/block.json

```diff
@@ -8,6 +8,9 @@
 	"keywords": [ "products", "summary", "excerpt" ],
 	"textdomain": "default",
 	"attributes": {
+		"name": {
+			"type": "string"
+		},
```

Comment:
Yeah `property` may indeed be the better one to prevent with `name` being used for something else.
My only slight push back is that `name` is used for the native input fields and most form libraries, but I do see how this can be a bit of an exception.

Reply:
Yep, I do agree with that, native input uses name to refers to an object prop and under the hood we should also use name in any of our form fields.

Reply:
With native HTML input fields, the `name` property is not really the name of an object property it is bound to. It is the name of the field, and that happens to also get used to send the value of the form field to the server on a form submission.

`name` is vague and confusing, since we also have the block `name` (`woocommerce/product-summary`which is also poorly chosen, it should be `type` in my opinion). 

Especially with the support Nathan has been working on to support meta data as well, we might have property paths, like `meta_data.foo` bound to blocks in the future.

Reply:
> and that happens to also get used to send the value of the form field to the server on a form submission.

You said better than me.

Reply:
Yup, makes sense. I did update the `summary` field to adhere to this.